torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -64,6 +64,7 @@ const auto& intListAttr = Symbol::attr("profiled_int_list");
 const auto& intAttr = Symbol::attr("profiled_int");
 const auto& boolListAttr = Symbol::attr("profiled_bool_list");
 const auto& boolAttr = Symbol::attr("profiled_bool");
+const auto& strAttr = Symbol::attr("profiled_str");
```

Comment:
nitpick. I should have cleaned this and move these Symbol to somewhere maybe utils.h and reuse that across graph_fuser.cpp e.t.c.